cyphers.py

```diff
@@ -41,18 +58,23 @@ def letter_frequency(c):
 elif mode == 'd':
     message1 = input('What is the message you want to decrypt? ')
     listmessage1 = list(message1)
-    freqall = []
-    for i in range(26):
-        shift = i
-        decrypt = new_message(listmessage1, encription, shift)
-        frequency = []
-        (map(letter_frequency, decrypt))
-        for i in range(len(decrypt)):
-            frequency.append(letter_frequency(decrypt[i]))
-        freqall.append(sum(frequency))
-    for x in freqall:
-        if x == max(freqall):
-            print(''.join(decrypt))
+
+    decrypted_strings = []
```

Comment:
First: generate the "decrypted" list of characters for each of the 26 possible shift values. We will use this later.